parcellab/microservice/values.yaml

```diff
@@ -183,3 +183,36 @@ postgresql:
 redis:
   enabled: false
   # See default values: https://artifacthub.io/packages/helm/bitnami/redis?modal=values
+
+#  these containers share secrets, configmaps, volumes and environment variables from the settings above
+initContainers: #[]
```

Comment:
```suggestion
initContainers: []
```

Reply:
I do understand though that you might want to skip setting initContainers at all. But it is most probably a [] by default in Kubernetes (and not an undefined).
In any case, for consistency purposes I would define the list in values. You can also decide if you remove the if condition for initContainers or not (as it won't be necessary if we enforce [])